dbt/adapters/redshift/connections.py

```diff
@@ -150,7 +150,7 @@ class RedshiftCredentials(Credentials):
     sslmode: Optional[UserSSLMode] = field(default_factory=UserSSLMode.default)
     retries: int = 1
     region: Optional[str] = None  # if not provided, will be determined from host
-    autocommit: Optional[bool] = False
+    deactivate_autocommit: Optional[bool] = False
```

Comment:
What was the rationale behind changing the argument to `deactivate_autocommit` instead of changing the default to `True`? Will this change what the users need to put in their profile?

Reply:
We want flags to be False by default. Since this flag would turn off the default behavior, the name has to be flipped.

And it will only change for those who used my janky workaround in the past two weeks. I'll be updating everyone on relevant issues

Reply:
What would be the consequences if we switch to `autocommit: Optional[bool] = True`?

If we leave it as `deactivate_autocommit`, what would be the affect on anyone that has been using the workaround?

`autocommit` is both shorter and more intuitive than `deactivate_autocommit`, so we should stick with it unless there's a really strong argument against it.

Reply:
We had an off-line conversation around this. Even if it's not intuitive, this is our convention of profile flags being often.

Reply:
After further discussions, we're gonna move ahead with autocommit with default as true, but we have further investigations internally on how we should approach problems like this in the future.